service/storage_service.hh

```diff
@@ -781,8 +781,12 @@ private:
     std::optional<shared_future<>> _decomission_result;
     std::optional<shared_future<>> _rebuild_result;
     std::unordered_map<raft::server_id, std::optional<shared_future<>>> _remove_result;
+    struct {
+        raft::term_t term{0};
+        size_t last_index{0};
+    } _raft_topology_cmd_handler_state{};
```

Comment:
one `{}` is enough - either at `_raft_topology_cmd_handler_state` or at `term` and `last_index` both produce zero initialisation, but is it really important?

Reply:
> but is it really important?

Of course not. It is like complaining about double ';'. If you need to produce a new version of the series anyway drop one, but otherwise no need to bother.

Reply:
`nit:` tag could help to make this intention clear